webpack.config.js

```diff
@@ -11,6 +15,21 @@ module.exports = {
 		filename: "browser-bundle.js"
 	},
 	plugins: [
-		new LicenseWebpackPlugin()
+		new LicenseWebpackPlugin({
+			addBanner: true,
+			unacceptableLicenseTest: (licenseType) => {
+				// Fail when not allowed licenses would be packaged
+				return !allowedLicenses.includes(licenseType);
+			},
+			renderLicenses: (modules) => {
+				const line = (v) => v ? `\n${v}` : "";
+				return modules.sort((a, b) => a.name < b.name ? -1 : 1).map((module) => {
+					return `${module.name} (${module.packageJson.version})${
```

Comment:
Can you reformat this a little bit. It's hard to read

e.g. 
```javascript
`${module.name} (${module.packageJson.version})
  ${line(module.packageJson.homepage)}
  ${line(module.licenseId)}
  ${line(module.licenseText)}`
```


```javascript
.sort(...)
.map(module => `...`) 
.join("\n\n");
```
In the map statement only one value (template literal) is returned. Perhaps it would make sense to remote the return statement (like in the line function on line 25)

Reply:
> ```
> `${module.name} (${module.packageJson.version})
>  ${line(module.packageJson.homepage)}
>  ${line(module.licenseId)}
>  ${line(module.licenseText)}`
> ```

There is a reason for the current formatting. It prevents new lines / tabs to be included. But I agree, it's hard to read.

What about this?
```js
let text = `${module.name} (${module.packageJson.version})`;
text += line(module.packageJson.homepage);
text += line(module.licenseId);
text += line(module.licenseText);
return text;
```
More verbose but easier to understand.

Reply:
Done